src/towncrier/_git.py

```diff
@@ -1,12 +1,24 @@
 # Copyright (c) Amber Brown, 2015
 # See LICENSE for details.
 
-from subprocess import call
-
 import os
+import sys
+from subprocess import call, check_output, CalledProcessError
+
 import click
 
 
+def run(args, **kwargs):
+    # Use UTF-8 both when sys.stdout does not have .encoding (Python 2.7) and
```

Comment:
Maybe have the comment as docstring ? 

And this is not really related to the git code.

Maybe keep is as the private `check._run` and update the private `_run` to always return decoded output.

Make sense?

Reply:
Yeah, but I need to call this function from the `remove_files` function, and it feels a bit weird to import a private method from the `check` module (which is a CLI module).
How would you do it?